legal-api/tests/unit/models/test_legal_entity.py

```diff
@@ -675,56 +678,30 @@ def test_legal_name_firms_SP(session, test_name, partner_info, expected_legal_na
 
 
 @pytest.mark.parametrize(
-    "entity_type, legal_name, operating_name, expected_business_name",
+    "entity_type, legal_name, expected_business_name",
     [
-        ("CP", "CP Test XYZ", None, "CP Test XYZ"),
-        ("BEN", "BEN Test XYZ", None, "BEN Test XYZ"),
-        ("BC", "BC Test XYZ", None, "BC Test XYZ"),
-        ("ULC", "ULC Test XYZ", None, "ULC Test XYZ"),
-        ("CC", "CC Test XYZ", None, "CC Test XYZ"),
-        ("SP", None, "SP Test XYZ", "SP Test XYZ"),
-        ("GP", None, "GP Test XYZ", "GP Test XYZ"),
+        ("CP", "CP Test XYZ", "CP Test XYZ"),
+        ("BEN", "BEN Test XYZ", "BEN Test XYZ"),
+        ("BC", "BC Test XYZ", "BC Test XYZ"),
+        ("ULC", "ULC Test XYZ", "ULC Test XYZ"),
+        ("CC", "CC Test XYZ", "CC Test XYZ"),
     ],
 )
-def test_business_name(session, entity_type, legal_name, operating_name, expected_business_name):
+def test_business_name(session, entity_type, legal_name, expected_business_name):
```

Comment:
Simplified by splitting the SP/GP specific test parameters into `test_alternate_name.py`